faq/index.md

```diff
@@ -3,11 +3,15 @@ layout: page
 title: Frequently Asked Questions
 ---
 
-## Why is JSON API not versioned? <a href="#why-is-json-api-not-versioned" id="why-is-json-api-not-versioned" class="headerlink"></a>
+## What is the meaning of JSON API's version? <a href="#what-is-the-meaning-of-json-apis-version" id="what-is-the-meaning-of-json-apis-version" class="headerlink"></a>
 
-Once JSON API is stable, it will always be backwards compatible using a _never
-remove, only add_ strategy.
-[#46](https://github.com/json-api/json-api/issues/46)
+Now that JSON API has reached a stable version 1.0, it will always be
+backwards compatible using a _never remove, only add_ strategy.
```

Comment:
I'm interpreting this as "we're going to follow semver, and we don't ever plan on making a version 2.0."